deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
to start a discussion around this... should we be updating the uuid_migrate script? we were only running it to get users back up to the state where their indices have a UUID in them.  I think we should discuss if we can use an index template alias instead for all other indices as we move forward